posts/2023/sdk_12_2_0_ga.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Titanium SDK 12.2.0.GA released'
+date: '2023-09-15'
+category: 'Release'
+author: 'Hans Knöchel'
+teaser: 'Stable iOS 17 and Xcode 15 support'
+---
+
+The final version of the next Titanium SDK is available: <b>12.2.0.GA</b>. Titanium SDK 12.2.0 is a minor release of the SDK, adding new features as well as
+addressing high-priority issues from previous releases. Highlights of this release are:
+
+- Full support for iOS 17 and Xcode 15
+- New Android APIs
+- iOS stability improvements
+
+For a more detailed overview, known issues and closed tickets, see the release notes: [Titanium SDK 12.2.0.GA Release Note](https://titaniumsdk.com/guide/Titanium_SDK/Titanium_SDK_Release_Notes/Titanium_SDK_Release_Notes_12.x/Titanium_SDK_12.2.0.GA_Release_Note.html).
+
+## Install
+
+**Follow these steps to get SDK 12.2.0.GA:**
+
+1. Install the CLI with `[sudo] npm i -g titanium alloy`
+2. Run `titanium sdk install 12.2.0.GA`
+3. Set `<sdk-version>12.2.0.GA</sdk-version>` in your `tiapp.xml`
+
+## Rollback
+
+Execute `titanium sdk install latest --default` to rollback.
+
```

Comment:
```suggestion
```

Remove this as `latest` will be the same version. Don't think we need that in the GA notes